doc/install.rst

```diff
@@ -0,0 +1,29 @@
+Installation
+===============================================
+
+
+To install go to the main folder and do:
+
+```pip install .```
+
+
+Requirements for users
```

Comment:
I would rather not list the requirements here, as this is likely to get outdated. I.e. requirements only listed in setup.py. / requirements.txt / setup.cfg / wherever it is.

Reply:
Makes sense :)
Just added a tiny instruction subsection to the main page and removed the install page.